libraries/webInterface/gui/index.js

```diff
@@ -1935,13 +1936,24 @@ realityServer.toggleFullScreen = function (item) {
         document.body.appendChild(thisIframe);
     }
 
-    let screenPort = realityServer.objects[item.id.slice('fullscreen'.length)].screenPort;
-    thisIframe.src = 'http://' + realityServer.states.ipAdress.interfaces[realityServer.states.ipAdress.activeInterface] + ':' + screenPort;
-
     let thisScreen = thisIframe;
     // if(item) thisScreen = item;
 
     if (!thisScreen.mozFullScreen && !document.webkitFullScreen) {
+        thisIframe.src = "about:blank"; // Clear iframe before loading
+        const targetUrl = `/obj/${item.dataset.objectName}/target/target.jpg`;
+        const iframeContents = `<div style="text-align: center;"><div style="background: url(${targetUrl}) no-repeat center; background-size: contain; height: 100%; width: 100%;"></div></div>`;
```

Comment:
We should talk about the intended behavior before merging. This button was originally linked to a special hardware interface that I built which let you push and pull AR content in/out of screens. Opening the iframe whose src was set to the object's "screenPort" would open a webpage hosted at that endpoint which provided the capabilities for doing so. I need to fix a few things with that feature before including it again. For now, just showing the image is ok, but we will have to change it sometime in the future.